jenkins-pipelines/perf-regression-latency-nosqlbench-mixed.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_nosqlbench.PerformanceRegressionNosqlBenchTest",
+    test_config: "test-cases/performance/perf-regression-nosqlbench-latency-mixed-50r-50w.yaml",
+    sub_tests: ["test_mixed"],
```

Comment:
Need to change that now that you changed the name.

Reply:
Fixed